association-backend/src/webhooks/handlers/pretix/api.py

```diff
@@ -0,0 +1,38 @@
+from typing import Literal
+
+import httpx
+
+from src.association.settings import PRETIX_API_TOKEN, PRETIX_API_URL
+
+METHODS = Literal["get"]
+
+
+class PretixAPI:
```

Comment:
this will soon live in toolkit and be reused between pycon backend and association: https://linear.app/python-italia/issue/WEB-211/muovere-linterfaccia-api-pretix-in-toolkit